packages/documentation/src/pages/platform/architecture/cms.mdx

```diff
@@ -45,6 +45,40 @@ Drupal is queried in the Jenkins build process for any vets-website branch. On J
 - [Jenkins build process flow](/platform/architecture/build-deploy-flows#standard-masterpr-build)
 - [Drupal/AWS cache script](https://github.com/department-of-veterans-affairs/vets-website/tree/master/script/drupal-aws-cache.js)
 
+### Automated accessibility checking
+
+All pages created and managed with the content management system are required to pass accessiblity scans for [WCAG2 Level A and AA](https://www.w3.org/TR/WCAG20/) compliance. The Jenkins build process automatically scans every page, and will break on Section 508 or WCAG violations. To help prevent this, we have [created an accessibility check](https://github.com/department-of-veterans-affairs/vets-website/pull/10836) that can be used on `localhost` and the CMS preview servers.
+
+#### Reviewing issues on localhost
+
+1. Start your SOCKS proxy
+2. Open a terminal window and type the following command except the dollar sign:
+3. `$ npm run build -- --pull-drupal --entry=static-pages --accessibility`
+4. When Step 3 is complete, type the following command:
+5. `$ npm run preview`
+6. Open a browser window and enter URL **http://localhost:3001/preview?nodeId=67**
+7. Click on the light blue banner at the top of the screen to learn more about any accessibility issues
+
+#### Reviewing issues on a preview server
+
+1. Start your SOCKS proxy
+2. Append `/preview?nodeId=<UNIQUE_ID>` to your preview environment URL in a new browser window
+3. Click on the light blue banner at the top of the screen to learn more about any accessibility issues
+
+#### Interpreting accessibility violations and warnings
+
+The accessibility preview shows Section 508 and WCAG violations, as well as best practice warnings. The Jenkins build process will break on accessibility violations, but not best practice warnings.
```

Comment:
this might be better stated in terms of required reviews. content edits go through our continuous integration process which has automated reviews for content changes. 508 violations will cause the automated review to fail.

I think it's fine to mention Jenkins but I don't expect content editors to know what it means.